### ISSUE TITLE
Netplan: Improve 'How-to contribute to docs'

### ISSUE THREAD

**Issue by rkratky** [canonical/open-documentation-academy#122]
## Context

While the Netplan docs include a how-to article with guidance on contributing to the documentation set, it could be improved by:

- Making it better findable
- Adding info about things one needs to know when submitting PRs to the Netplan repo

## Task

1. Move the [Contribute to docs](https://canonical-netplan.readthedocs-hosted.com/en/latest/contribute-docs/) article from the How-to section in the navigation to the top level, i.e. as a standalone link under the four Diátaxis categories in the left-hand sidebar.
2. Add a section to this how-to that would prepare the contributor for what's required when submitting a Netplan PR:
   - Short explanation of tests that need to be run in the repo's main directory
   - Prerequisites needed for the test suites to function

## Reference

- Diataxis: [How-to guides](https://diataxis.fr/how-to-guides/)
- ReadTheDocs: [Netplan documentation](https://netplan.readthedocs.io/)
- GitHub: [netplan](https://github.com/canonical/netplan/tree/main/doc)

**Comment by davidekete** [comment]
Hi @rkratky, I'd like to work on this

**Comment by rkratky** [comment]
Nice. Just assigned the issue to you.

**Comment by davidekete** [comment]
Hi @rkratky, I just noticed another issue with the contribute to docs tutorial. It [mentions](https://netplan.readthedocs.io/en/stable/contribute-docs/#directory-structure) that "All the documentation files are located in the doc/ directory. The doc/ directory contains sub-directories corresponding to different [Diátaxis](https://diataxis.fr/) sections". This statement is wrong, the doc/ directory does not contain any sub-directories corresponding to different Diátaxis sections.

**Comment by rkratky** [comment]
Right, that's an oversight -- using the `doc/` directory is the usual set up, and we forgot this was not the case for Netplan when adopting the contrib. guide.

OK, I'm creating ano0ther issue to deal with that particular discrepancy. Please, ignore that part while working on this issue.

**Comment by davidekete** [comment]
> Right, that's an oversight -- using the `doc/` directory is the usual set up, and we forgot this was not the case for Netplan when adopting the contrib. guide.
> 
> OK, I'm creating ano0ther issue to deal with that particular discrepancy. Please, ignore that part while working on this issue.

Alright, please let me know when you create the issue so you can assign it to me.

**Comment by rkratky** [comment]
https://github.com/canonical/open-documentation-academy/issues/125
